client/posts/render/body.ts

```diff
@@ -256,17 +255,31 @@ function parseReds(
         parseBlues(frag, state, fn)
     let html = ""
 
-    while (true) {
-        const i = frag.indexOf("^r")
-        if (i !== -1) {
-            html += _fn(frag.slice(0, i)) + wrapTags(3, state)
-            state.red = !state.red
-            frag = frag.substring(i + 2)
-        } else {
-            html += _fn(frag)
-            break
+    if (boardConfig.rbText) {
+        while (true) {
+            const i = frag.indexOf("^r")
+            if (i !== -1) {
+                html += _fn(frag.slice(0, i)) + wrapTags(3, state)
```

Comment:
Why not put `wrapTags(3, state)` and `state.red = !state.red` behind an `if (boardConfig.rbText) {` instead of this dumplication?